7/s2i/bin/usage

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+cat <<EOF
+This is an S2I for LLVM $LLVM_VERSION
+To use it, install S2I: https://github.com/openshift/source-to-image
+
+You will need to add your own assemble / run scripts to $srcdir/.s2i/bin
+for building and running your code.   Here is an example assemble script
+
+################################################################################
+# Example assemble script to be placed in $srcdir/.s2i/bin/assemble
+################################################################################
```

Comment:
If I'm not mistaken, `usage`  is outputted when the container is run without build, IOW a sort of `--help`.
Do we really want to output sample config files there?
I'd rather see the project README.md, or just the assemble file, linked via url.
WDYT? (cc: @hhorak )

Reply:
I don't have a strong opinion here, but @pvalena is correct that the usage is mostly similar to `--help` output.